spec/policy_machine_storage_adapters/active_record_spec.rb

```diff
@@ -656,6 +708,105 @@
         end
       end
 
+      describe 'accessible_objects_for_operations_function' do
+        before do
+          expect_any_instance_of(PolicyMachineStorageAdapter::ActiveRecord)
+            .to receive(:accessible_objects_for_operations_function)
+            .and_call_original
+        end
+
+        context 'when there are directly accessible objects' do
+          before do
+            priv_pm.add_association(color_1, creator, object_6)
+            priv_pm.add_association(color_2, creator, object_7)
+          end
+
+          it 'returns objects accessible via each of multiple given operations' do
+            result = priv_pm.accessible_objects_for_operations(
```

Comment:
i would normally do this `result` local as a `let` and put these examples in `context` blocks and then set individual params in the `let`. 
that said i feel fairly ambivalent here and dont think u need to spend the extra time 😄